proto/dwtools/abase/l0.test/Buffer.test.s

```diff
@@ -206,6 +206,674 @@ function bufferNodeIs( test )
+  var got = _.bufferAnyIs( src );
+  var expected = true;
+  test.identical( got, expected );
+
+  test.case = 'check BufferRaw';
+  var src = new U8x( [ 1, 2, 3 ] ).buffer;
+  var got = _.bufferAnyIs( src );
+  var expected = true;
+  test.identical( got, expected );
+
+  test.case = 'check BufferRawShared';
+  var src = new BufferRawShared( 5 );
+  var got = _.bufferAnyIs( src );
+  var expected = true;
+  test.identical( got, expected );
+
+  /* BufferView */
+
+  test.case = 'check BufferView';
+  var src = new BufferView( new U8x( [ 1, 2, 3 ] ).buffer );
+  var got = _.bufferAnyIs( src );
+  var expected = true;
+  test.identical( got, expected );
+
+  /**/
+
+  if( !Config.debug )
+  return;
+
+  test.case = 'check without argument';
+  var got = _.bufferAnyIs();
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check extra arguments';
+  var got = _.bufferAnyIs( new U8x( [ 1, 2, 3 ] ), new BufferRaw( 5 ) );
+  var expected = true;
+  test.identical( got, expected );
+}
+
+//
+
+function bufferBytesIs( test )
+{
+
+  test.case = 'check null';
+  var got = _.bufferBytesIs( null );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check undefined';
+  var got = _.bufferBytesIs( undefined );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check zero';
+  var got = _.bufferBytesIs( 0 );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check false';
+  var got = _.bufferBytesIs( false );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check NaN';
+  var got = _.bufferBytesIs( NaN );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check Symbol';
+  var got = _.bufferBytesIs( Symbol() );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check empty map';
+  var got = _.bufferBytesIs( {} );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check empty array';
+  var got = _.bufferBytesIs( [] );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check string';
+  var got = _.bufferBytesIs( 'string' );
+  var expected = false;
+  test.identical( got, expected );
+
+  /* BufferTyped */
+
+  test.case = 'check empty U8x';
+  var src = new U8x( 5 );
+  var got = _.bufferBytesIs( src );
+  var expected = true;
+  test.identical( got, expected );
+
+  test.case = 'check empty U16x';
+  var src = new U16x( [ 2, 3 ] );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check empty U32x';
+  var src = new U32x( 5 );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check empty U64x';
+  var src = new U64x( 2 );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  /**/
+
+  test.case = 'check U8x';
```

Comment:
Please, remove test group. It's copy of previous test group.

For this test routine use one instance of constructor per class.
The full list of classes

- `I8x`
- `U8x`
- `U8ClampedX`
- `I16x`
- `U16x`
- `I32x`, similar to the name `Ix`
- `U32x`, similar to the name `Ux`
- `F32x`, similar to the name `Fx`
- `F64x`
- `I64x`
- `U64x` 

Use different instances, empty, filled and not filled